.github/workflows/unit-tests.yaml

```diff
@@ -28,8 +28,19 @@ jobs:
     steps:
     - name: Checkout msamp
       uses: actions/checkout@v2
+      with:
+          submodules: true
+    - name: Install nccl
+      run: |
+        cd third_party/nccl
+        make -j src.build NVCC_GENCODE="\
+          -gencode=arch=compute_70,code=sm_70 \
+          -gencode=arch=compute_80,code=sm_80 \
```

Comment:
remove?

Reply:
There may be no need to remove them since it can accelerate the compilation of nccl.